lib/src/handle_access_token_response.dart

```diff
@@ -78,7 +81,7 @@ Credentials handleAccessTokenResponse(
   }
 
   var scope = parameters['scope'] as String;
-  if (scope != null) scopes = scope.split(" ");
+  if (scope != null) scopes = scope.split(delimiter ?? " ");
```

Comment:
You shouldn't need the `??` here, since all callers are passing in non-null delimiters